src/openrct2/ride/Track.h

```diff
@@ -237,6 +237,7 @@ enum class TrackCurve : uint8_t
     RightSmall = 4,
     RightVerySmall = 6
 };
+constexpr const uint8_t kHighestCurveValue = 8;
```

Comment:
This name is not very intuitive to me, tbh. Does it refer to track tile elements, or something else entirely?

Reply:
It refers to TrackCurve, which is specified right above it.

Reply:
Ah, I didn't see that due to the diff being cut off. That clears it up! Never mind then.